packages/grafana-ui/src/components/uPlot/config/UPlotConfigBuilder.ts

```diff
@@ -29,6 +29,8 @@ const cursorDefaults: Cursor = {
 type PrepData = (frames: DataFrame[]) => AlignedData | FacetedData;
 type PreDataStacked = (frames: DataFrame[], stackingGroups: StackingGroup[]) => AlignedData | FacetedData;
 
+type PlotState = { isPanning: false } | { isPanning: true; min: number; max: number };
```

Comment:
I like this!

Reply:
Thanks! 😄 I wanted it to be a little more strict than `?`s for `min` and `max`.